actions/lib/prepare_archive_package.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Simple script that should be executed on a biotank when we want to prepare a
+# runfolder to be archived to PDC via TSM.
+
+set -e
+
+# The full path to the runfolder that we want to archive.
+RUNFOLDER=$1
+
+RUNFOLDER_NAME=$(basename ${RUNFOLDER})
+
+# The exclude pattern is configurable upstreams, but it should hopefully look
+# something like the following. Escaping various quotes can be messy though. 
+# EXCLUDE="^./Config|^./InterOp|^./SampleSheet.csv|^./Unaligned|^./Data|^./Thumbnail_Images"
+EXCLUDE=$2
+
+cd ${RUNFOLDER}
+
+# TODO: Shall we upload it when it exists?
```

Comment:
Perhaps we should ask a force flag of some kind here so that as little as possible manual work is required to do this?